crates/optimism/payload/src/builder.rs

```diff
@@ -365,13 +371,30 @@ where
             }
         }
 
-        let withdrawals_root = ctx.commit_withdrawals(state)?;
+        debug_assert!(ctx.attributes().payload_attributes.withdrawals.is_empty());
```

Comment:
I don't think we should assert this because this is external input

Reply:
true specs actually doesn't mention this is empty